Backend/TimeTracker.Discord/Commands/TimeEntryCommands.cs

```diff
@@ -20,9 +20,19 @@ public TimeEntryCommands(CommandService commandService, MainDb database)
             this.commandService = commandService;
         }
 
+        [Command("test")]
+        [Summary("Create a time entry for the current day.")]
+        public async Task TestCall(int projectId, decimal length, string notes)
+        {
+            await Context.Message.ReplyAsync(projectId.ToString());
+            await Context.Message.ReplyAsync(length.ToString());
+            await Context.Message.ReplyAsync(notes);
+            return;
+        }
+
         [Command("create")]
         [Summary("Create a time entry for the current day.")]
-        public async Task Create([Remainder] string parameters)
+        public async Task Create(int? projectId, double? length, [Remainder] string notes = "")
```

Comment:
Still using the projectId

Reply:
Since there are spaces in some project names, we have to do projectIds in order to specify the project